pkg/sourcereader/modules/test_data/network/vpc/README.md

```diff
@@ -0,0 +1,50 @@
+<!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
```

Comment:
You could exclude this test module from terraform docs at [this line](https://github.com/thiagosgobe/hpc-toolkit/blob/sourcereader-wrapfs-workaround/.pre-commit-config.yaml#L32) of the precommit config.

Something like the following should work (not fully tested!):
`exclude: \[.terraform|pkg]\/.*$`

This would exclude any path in pkg, therefore anything in the go code.

When you do that, you can also remove this file.

Reply:
It works like that:

`exclude: \.terraform|pkg\/.*$`